### ISSUE TITLE
Ubuntu 17.10

### ISSUE THREAD

**Issue by samiball** [janbar/noson-app#18]
#Ilovethhisapp

Hi, I just tested the application on Ubuntu 17.10, and unfortunately it does not run. Would you know why?
thank you in advance for your help

**Comment by janbar** [comment]
Yes I know. The cause is "Ubuntu qml components" package has been removed in 17.10. unfortunately the app depends of it.
But the SNAP package works. So you could try the snap instead legacy package.
In future, when I will have some time I will refactor the UI without ubuntu components.

**Comment by janbar** [comment]
This is a pity because all apps designed for ubuntu touch since 15.04, are now broken !

**Comment by samiball** [comment]
Yes really bad. Good luck to you...
For information, the SNAP package does not work either
snapd 2.28.5+17.10 with sonos 2.4.12-SNAP1

**Comment by samiball** [comment]
Hi 
I confirm that everything works with the SNAP package
@janbar thanks a lot for your help

**Comment by janbar** [comment]
I refactored the app for Ubuntu 17.10 and later. No SNAP package is available for this version. You have to install the deb package from my PPA.
A SNAP package could be available once 18.04 is out (April 2018). But you should prefer the native package from my PPA.
